assets/setup-wizard/style.scss

```diff
@@ -328,10 +328,17 @@
 		.woocommerce-list__item-after {
 
 			.components-button {
-				width: 160px;
+				width: 205px;
 				margin-left: 10px;
 				justify-content: center;
 			}
+
+			.sensei-setup-wizard__error-message {
```

Comment:
A small visual suggestion, this could use a bit of spacing, like a `padding: 6px`

Reply:
Do you mean all the sides? Or on the top only?

Reply:
Yup all sides should be fine.

Reply:
Added the spacing and updated the PR description 😊  https://github.com/Automattic/sensei/pull/3507/commits/fd24a5ecdb7b0fe992644c13bfb60434c2a63f95